### ISSUE TITLE
Coreograph needs more cores when writing cores on O2

### ISSUE THREAD

**Issue by clarenceyapp** [labsyspharm/mcmicro#65]
@ArtemSokolov  In the config file, please add more CPU cores on O2 to Coreograph so that it saves TMA cores in parallel. For example, add -c 10. Doing so will also reduce the time needed from 1 hour down to 30 minutes.

**Comment by ArtemSokolov** [comment]
Sounds like a duplicate of #40 ?

**Comment by clarenceyapp** [comment]
Yes, Let me know if the proposed change isn't going to work with 10 cores. Closing this issue.
